### ISSUE TITLE
getting Episodes list in IMDB seems to be not working

### ISSUE THREAD

**Issue by copperhead57** [andig/videodb#89]
when trying to display or change the episode listing in imbd  brings back none of the episodes. this occurrs on all series i have tried

**Comment by kec2** [comment]
@copperhead57 Could you please try the latest version and report back

**Comment by copperhead57** [comment]
@kec2 , have tried fix. but hasn't fixed issue, here is more detail as may not have explained issue clearly.
select imdb online
in search bar - walking dead - click search icon
select The Walking Dead (2010) (TV Series) 
under episodes year click see all - episodes are listed
in year select 2016 - no episodes are listed

With the supplied fix no title or sub title was saved.

**Comment by kec2** [comment]
I have created a fix for the missing title, subtitle and year.

But there seams to be a lot wrong with IMDB online.
There are lots of broken links to images.

in search bar - typing "walking dead" - clicking on an option in the drop down box -> Not found error page.

In the episodes list - changing season does not change anything.

At first I was able to replicate the problem regarding changing year in the episodes list. But later nothing happened.
